apps/frontend/src/app/3_organisms/ZeroLocForm/types.ts

```diff
@@ -1,14 +1,17 @@
 import { SupportedTokenList, SupportedTokens } from '@sovryn/contracts';
 import type { SelectOption } from '@sovryn/ui';
 
+import { BORROW_ASSETS } from '../../5_pages/ZeroPage/constants';
+
 export enum AmountType {
   Add = 'Add',
   Remove = 'Remove',
 }
 
-export const tokens: SelectOption<SupportedTokens>[] = SupportedTokenList.map(
-  token => ({
-    value: token.symbol,
-    label: token.symbol.toUpperCase(),
-  }),
-);
+export const tokens: SelectOption<SupportedTokens>[] =
```

Comment:
Use `tokensToOptions` method from `utils` instead (in the component). Also, extract `tokensToOptions` globally so we can use it throughout the dapp.